modules/@apostrophecms/login/ui/apos/mixins/AposLoginPasswordResetEnabledMixin.js

```diff
@@ -0,0 +1,9 @@
+// Provides enabled computed status of the password reset feature
+
+export default {
+  computed: {
+    passwordResetEnabled() {
```

Comment:
I think this is overkill because it can never change during the lifetime of the webpage. Should just go ahead and reference `apos.login.passwordResetEnabled`